lib/msal-browser/docs/logging.md

```diff
@@ -59,5 +59,26 @@ An example usage in a sample can be accessed [here](https://github.com/AzureAD/m
 
     ![browser console](./images/BrowserLogEnablement.png)
 
+## Override log level and PII setting
+
+These are the steps to override MSAL log level and PII settings to troubleshoot errors in non-dev environments:
+
+### Navigate to local storage
+
+1. Open developer tools by pressing F12
+2. Navigate to `Application` tab
+3. Click `Storage` and expand `Local Storage`
```

Comment:
Why localStorage rather than sessionStorage?

Reply:
Updated to session storage.